cmd/hstspreload/format.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+const (
+	reset  = "\033[0m"
```

Comment:
I'd rename this to something like colorReset or styleReset. It's used a couple of times in the code and is slightly confusing with a short name (sounds like a bool instead of a string).